<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSchemesWindow</class>
 <widget class="QMainWindow" name="ColorSchemesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>916</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro Black</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HSV</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro Black</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CMYK</string>
    </property>
   </widget>
   <widget class="QSlider" name="hueSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="saturationSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="valueSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="yellowSlider">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>160</width>
      <height>10</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="cyanSlider">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>160</width>
      <height>10</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="magentaSlider">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>160</width>
      <height>10</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="blackSlider">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>160</width>
      <height>10</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="bgLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>431</width>
      <height>911</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>361</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>620</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>650</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
   <zorder>bgLabel</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>hueSlider</zorder>
   <zorder>saturationSlider</zorder>
   <zorder>valueSlider</zorder>
   <zorder>yellowSlider</zorder>
   <zorder>cyanSlider</zorder>
   <zorder>magentaSlider</zorder>
   <zorder>blackSlider</zorder>
   <zorder>label_3</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
